clusterloader2/pkg/test/interface.go

```diff
@@ -37,5 +37,5 @@ type TestExecutor interface {
 	ExecuteTest(ctx Context, conf *api.Config) error
 	ExecuteStep(ctx Context, step *api.Step) error
 	ExecutePhase(ctx Context, phase *api.Phase) error
-	ExecuteObject(ctx Context, object *api.Object) error
+	ExecuteObject(ctx Context, namespace string, replicaCount int32, object *api.Object) error
```

Comment:
Could you move these two parameters to the right? (just to make the first 2 args consistent across all these functions)

Reply:
OK.